libs/ui/common/src/index.ts

```diff
@@ -7,6 +7,11 @@ export const DropdownDialog = {
 export * from "./lib/author/author-chip";
 export * from "./lib/alert/alert";
 export * from "./lib/button/button";
+export * from "./lib/button/delete-button";
+export * from "./lib/button/add-button";
+export * from "./lib/button/edit-button";
+export * from "./lib/button/universal-button";
```

Comment:
hier sollten nur dateien eingetragen werden, die auch wirklich für andere libraries (oder die app selber) gedacht sind.

Reply:
Die Werden Gerade alle in der App genutzt.